index.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Porfolio</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <nav>
+      <p>Toyin</p>
+      <ul>
+        <li><a href="">About</a></li>
```

Comment:
This and other occurrences should be bookmarks. Only 1 page site is needed. Refer to https://www.w3schools.com/html/html_links_bookmarks.asp